frontend/app/routes/$lang/_public/apply/$id/child/children/$childId/dental-insurance.tsx

```diff
@@ -130,7 +130,7 @@ export default function AccessToDentalInsuranceQuestion() {
       </ul>
       <Collapsible summary={t('children.dental-insurance.detail.additional-info.title')}>
         <div className="space-y-4">
-          <p>{t('dental-insurance.detail.additional-info.eligible')}</p>
+          <p>{t('children.dental-insurance.detail.additional-info.eligible')}</p>
```

Comment:
I think this fix can be applied to "adult-child" children flow.

Reply:
This has not been addressed yet on adult-child file `frontend\app\routes\$lang\_public\apply\$id\adult-child\children\$childId\dental-insurance.tsx`

![image](https://github.com/DTS-STN/canadian-dental-care-plan/assets/114004123/e8c1148f-cabe-4f63-b72b-1b61e8b6e622)